pkg/services/store/object/http.go

```diff
@@ -47,21 +52,26 @@ func (s *httpObjectStore) RegisterHTTPRoutes(route routing.RouteRegister) {
 	route.Get("/history/*", reqGrafanaAdmin, routing.Wrap(s.doGetHistory))
 	route.Get("/list/*", reqGrafanaAdmin, routing.Wrap(s.doListFolder)) // Simplified version of search -- path is prefix
 	route.Get("/search", reqGrafanaAdmin, routing.Wrap(s.doSearch))
+
+	// File upload
+	route.Post("/upload", reqGrafanaAdmin, routing.Wrap(s.doUpload))
```

Comment:
Is this related to GRN switch or just adding a missing endpoint for storage story?

Reply:
This was a missing endpoint that i have been using to test the service in https://github.com/grafana/grafana/pull/56348